Codeinwp/neve
# Workflow Improvements | Front end resize events

### Summary
- removes bricks.js unused dependency
- adds resize event handling for dropdowns
- improves workflow - only need to use `npm` tasks for prebuilds and builds


### Will affect visual aspect of the product

NO


### Test instructions


- `$ npm install` 
- `$ npm run dev` and test that JS/CSS files are compiled realtime
- `$ npm run build` and check if the JS/CSS is built properly 

cc @cristian-ungureanu @rodica-andronache @selul
